dns/README.md

```diff
@@ -0,0 +1,26 @@
+# SSSL DNS server
+
+Custom DNS server intended to be used in conjunction with [SSSL](https://github.com/PretendoNetwork/SSSL). It redirects
+Nintendo hostnames to an SSSL-powered server using DNS spoofing.
+
+## Config
+
+The only 2 addresses required are for `conntest.nintendowifi.net` and `account.nintendo.net`. These can either be set
+using the default address or explicitly mapping them. Additional addresses may be added using `SSSL_DNS_MAP`.
```

Comment:
Why is this 2 lines?

Reply:
Hardwrapping. Didn't realize this was a problem, will fix it.